OEL-1418: Remove obsolete paragraphs requirements from oe_bootstrap_theme (#243)
openeuropa/oe_bootstrap_theme: README.md

## Upgrade from older versions

### Upgrade to 1.0.0-beta2

> The following paragraphs required components have been removed from [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme)

Reviewer: What is a "paragraphs required component"?

Reviewer: As the list below is about composer packages, we should say that:
```suggestion
Dependencies to the following packages have been removed in composer.json:
```
just a proposal, feel free to improve

* drupal/description_list_field
* openeuropa/oe_content
* openeuropa/oe_media
* openeuropa/oe_paragraphs

Be sure to require the component if you need any of them.

> 

Reviewer: Perhaps we should mention that you need to upgrade to alpha8 first if you had something older than that.

Author: Has sense? I'm reading the upgrade to 1.0.0-beta2 section, and the requirements to to the previous version is inherent...

Reviewer: The point is you will need a separate deployment.
Although, maybe this is only relevant once we fully remove the oe_bootstrap_theme_paragraphs module.

Reviewer: @drishu thoughts? ^

Reviewer: I tested the upgrade path from < alpha8.
There are problems, but they are fully covered by the comment that you already added here.
We will need to add another line later if/when we actually remove oe_bootstrap_theme_paragraphs.
So ok for now :)

Paragraphs-related theming and functionality has been moved from the [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme) to [OpenEuropa Whitelabel](https://github.com/openeuropa/oe_whitelabel).

If you are using `openeuropa/oe_whitelabel`, and you want paragraphs functionality, you should upgrade it to `1.0.0-alpha6` or higher, and refer to the upgrade instructions found there.

> 

Reviewer 2: above this section, add "Upgrade to 1.0.0-beta2"
there we explain that these composer packages are removed now and if needed, action needs to be taken.

Paragraphs-related theming and functionality has been moved from the [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme) to [OpenEuropa Whitelabel](https://github.com/openeuropa/oe_whitelabel).

If you are using `openeuropa/oe_whitelabel`, and you want paragraphs functionality, you should upgrade it to `1.0.0-alpha6` or higher, and refer to the upgrade instructions found there.

> ### Upgrade to 1.0.0-beta2

Reviewer 2: Please add this above the old one

> The following paragraph's required components has been removed from [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme)

Reviewer 2: Lets use plural here
The following paragraphs required components have been ...

  * drupal/description_list_field
  * openeuropa/oe_content
  * openeuropa/oe_media
  * openeuropa/oe_paragraphs

> If any of them are required, then you must install them like any other component outside oe_bootstrap_theme.

Reviewer 2: No, rather in the sorts of: Make sure to require them if you need them.
or something like that.
